.gitlab-ci.yml

```diff
@@ -739,9 +740,6 @@ workflow:
   - pkg/network/**/*
   - pkg/process/monitor/*
   - pkg/util/kernel/**/*
-  - test/kitchen/site-cookbooks/dd-system-probe-check/**/*
```

Comment:
This (and the following block) is not supposed to be here (it's already in main), should vanish if I do a rebase I guess (I'm waiting for the pipeline to run fully once first)